testing/test_pokemon.py

```diff
@@ -0,0 +1,63 @@
+"""Test module."""
+
+import pokemon.pokemon as p
+import unittest
+import pytest
```

Comment:
One wouldn't normally use both of these as far as I am aware? Just one or the other. From the way you've structured things below it looks as though pytest is the one you wane to keep.

Reply:
The main reason I included unittest was to use assertAlmostEqual. I need to find something similar in pytest.